src/main/flight/failsafe.c

```diff
@@ -63,27 +64,39 @@ failsafeState_t* failsafeInit(rxConfig_t *intialRxConfig)
     rxConfig = intialRxConfig;
 
     failsafeState.events = 0;
-    failsafeState.enabled = false;
+    failsafeState.monitoring = false;
 
     return &failsafeState;
 }
 
-bool failsafeIsIdle(void)
+failsafePhase_e failsafePhase()
 {
-    return failsafeState.counter == 0;
+    return failsafeState.phase;
 }
 
-bool failsafeIsEnabled(void)
+#define MAX_COUNTER_VALUE_WHEN_RX_IS_RECEIVED_AFTER_RX_CYCLE 1
```

Comment:
yeah, i had a hard time naming it too :smile: it was better than just '1' though.

at the start of the rx cycle the counter is increased.
if the counter had been reset due to valid data being received then the counter will be 0 at the end of the cycle.
if the counter was not reset the counter will either be 1, or the value of the counter from the previous iteration + 1.

perhaps 'MAX_COUNTER_VALUE_WHEN_RX_IS_RECEIVED_AFTER_RX_CYCLE_COMPLETES' is clearer?

Still way too long for my liking, i would like it be be succinct.  We can dwell on it, maybe something will spring to mind.